lib/google_drive/worksheet.rb

```diff
@@ -526,6 +547,142 @@ def inspect
+        fields << ",verticalAlignment"
+      end
+
+      fields << ")"
+      format_cells(start_row, start_col, end_row, end_col, format, fields)
+    end
+
+    # Change the background color on a range of cells. For example:
+    #   cell_background_color(1, 1, 1, 1, GoogleDrive::DARK_YELLOW_1)
+    def cell_background_color(start_row, start_col, end_row, end_col, background_color)
+      format = Google::Apis::SheetsV4::CellFormat.new
+      format.background_color = background_color
+
+      fields = "userEnteredFormat(backgroundColor)"
+      format_cells(start_row, start_col, end_row, end_col, format, fields)
+    end
+
+    # Change the text formatting on a range of cells.  For example, set cell
+    # A1 to have red text that is bold and italic:
+    #   text_format_cells(1, 1, 1, 1, true, true, false, nil, nil, GoogleDrive::RED_BERRY)
+    # Google API reference:
+    # https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets#textformat
+    def text_format_cells(start_row, start_col, end_row, end_col, bold = false,
+                          italic = false, strikethrough = false, font_size = nil,
+                          font_family = nil, foreground_color = nil)
+
+      text_format = Google::Apis::SheetsV4::TextFormat.new
+      format = Google::Apis::SheetsV4::CellFormat.new
+      format.text_format = text_format
+
+      text_format.bold = bold
+      text_format.italic = italic
+      text_format.strikethrough = strikethrough
+
+      unless font_size.nil?
+        text_format.font_size = font_size
+      end
+
+      unless font_family.nil?
+        text_format.font_family = font_family
+      end
+
+      unless foreground_color.nil?
+        text_format.foreground_color = foreground_color
+      end
+
+      fields = "userEnteredFormat(textFormat)"
+      format_cells(start_row, start_col, end_row, end_col, format, fields)
+    end
+
+    def format_cells(start_row, start_col, end_row, end_col, format, fields)
+      cell_data = Google::Apis::SheetsV4::CellData.new
+      cell_data.user_entered_format = format
+
+      request = Google::Apis::SheetsV4::RepeatCellRequest.new
+      request.range = v4_range_object(start_row, start_col, end_row, end_col)
+      request.cell = cell_data
+      request.fields = fields
+
+      @spreadsheet.add_to_batch_updates(repeat_cell: request)
+      @made_v4_changes = true
+    end
+
+    # Style options:
+    #   "DOTTED"  The border is dotted.
+    #   "DASHED"  The border is dashed.
+    #   "SOLID" The border is a thin solid line.
+    #   "SOLID_MEDIUM"  The border is a medium solid line.
+    #   "SOLID_THICK" The border is a thick solid line.
+    #   "NONE"  No border. Used only when updating a border in order to erase it.
+    #   "DOUBLE"  The border is two solid lines.
+    # Google API reference: https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets#Style
+    def google_border(style, color)
```

Comment:
Maybe more straightforward to simply alias Google::Apis::SheetsV4::Border to GoogleDrive::Worksheet::Border. Or maybe accept an array [style, color] instead of Border class.